modules/currency-l0/src/main/scala/org/tessellation/currency/l0/modules/Daemons.scala

```diff
@@ -24,7 +26,8 @@ object Daemons {
     queues: Queues[F],
     healthChecks: HealthChecks[F],
     maybeDataApplication: Option[BaseDataApplicationL0Service[F]],
-    config: AppConfig
+    config: AppConfig,
+    doubleSignDetectConfig: DoubleSignDetectConfig
```

Comment:
you already have or should have all the config above, in: `config: AppConfig`

Reply:
Moved all configuration for the daemon to `DoubleSignDetectDaemonConfig`, which are included in the various `AppConfig`s.